backend/src/middleware/formConfigLoader.ts

```diff
@@ -22,7 +22,7 @@ export const formConfigLoader = ({formConfigsDir}: {formConfigsDir: string}) =>
     try {
       const formId = req.params.formId;
       if (!formId) {
-        throw new GenericRequestError('No formId provided');
+        next(new GenericRequestError('No formId provided', 400, {errorCode: 'FORM_ID_MISSING'}));
```

Comment:
Do we still have to return afterwards? Otherwise the following code still runs, right?

Reply:
Correct, I've added this in [465c33b](https://github.com/formcapture/form-backend/pull/164/commits/465c33b219e56f3e7ea1f14ffd6017ffbe85d919)